SpeedTestTests/ReactiveSwiftTests.swift

```diff
@@ -0,0 +1,308 @@
+//
+//  ReactiveSwiftTests.swift
+//  SpeedTestTests
+//
+//  Created by Jakub Olejník on 08/08/2019.
+//  Copyright © 2019 QuickBird Studios. All rights reserved.
+//
+
+import XCTest
+import ReactiveSwift
+
+class ReactiveSwiftTests: XCTestCase {
+    
+    func testPublishSubjectPumping() {
+        measure {
+            var sum = 0
+            let subject = Signal<Int, Never>.pipe()
+            
+            let subscription = subject.output
+                .observeValues { x in
+                    sum += x
+            }
+            
+            for _ in 0 ..< iterations * 100 {
+                subject.input.send(value: 1)
+            }
+            
+            subscription?.dispose()
+            
+            XCTAssertEqual(sum, iterations * 100)
+        }
+    }
+    
+    func testPublishSubjectPumpingTwoSubscriptions() {
+        measure {
+            var sum = 0
+            let subject = Signal<Int, Never>.pipe()
+            
+            let subscription1 = subject.output
+                .observeValues { x in
+                    sum += x
+            }
+            
+            let subscription2 = subject.output
+                .observeValues { x in
+                    sum += x
+            }
+            
+            for _ in 0 ..< iterations * 100 {
+                subject.input.send(value: 1)
+            }
+            
+            subscription1?.dispose()
+            subscription2?.dispose()
+            
+            XCTAssertEqual(sum, iterations * 100 * 2)
+        }
+    }
+    
+    func testPublishSubjectCreating() {
+        measure {
+            var sum = 0
+            
+            for _ in 0 ..< iterations * 10 {
+                let subject = Signal<Int, Never>.pipe()
+                
+                let subscription = subject.output
+                    .observeValues { x in
+                        sum += x
+                }
+                
+                for _ in 0 ..< 1 {
+                    subject.input.send(value: 1)
+                }
+                
+                subscription?.dispose()
+            }
+            
+            XCTAssertEqual(sum, iterations * 10)
+        }
+    }
+    
+    func testMapFilterPumping() {
+        measure {
+            var sum = 0
+            
+            let subscription = SignalProducer<Int, Never> { observer, _ in
+                for _ in 0 ..< iterations * 10 {
+                    observer.send(value: 1)
+                }
+                observer.sendCompleted()
```

Comment:
Since we don't do this for the RxSwift and Combine tests I would suggest removing the .sendCompleted in the ReactiveSwift tests as well

Reply:
Done.